comps/reranks/README.md

```diff
@@ -0,0 +1,39 @@
+# ReRank Microservice
+
+## Introduction
+
+Provides functionality for reranking outputs.
+It allows for fine-tuning the results based on specific criteria, improving the overall quality and relevance for the user.
+
+## Key Features
+
+- **Configurable Ranking Criteria**: Define custom ranking rules based on various factors like relevance, creativity, style, or user-defined metrics.
+- **Multiple Ranking Models**: Integrate with different ranking models, enabling flexibility in selection based on the desired outcome and data type.
```

Comment:
the reranker cannot select the data type. The type of the instances define the output type